jina/peapods/container.py

```diff
@@ -69,13 +69,14 @@ def post_init(self):
         _args = kwargs2list(non_defaults)
         self._container = self._client.containers.run(self.args.uses, _args,
                                                       detach=True, auto_remove=True,
-                                                      ports={f'{v}': v for v in
+                                                      ports={'%d/tcp' % v: v for v in
```

Comment:
Please stick with f-strings: `f'{v}/tcp'.

Reply:
yes, my fault, it was just a conservative revert